src/sass/main.scss

```diff
@@ -94,3 +94,7 @@
 
 @import 'overrides/overrides.scss';
 @import 'overrides/print.scss';
+
+.pac-container {
```

Comment:
It's a bootstrap class, as I'm aware. If you remove it and try to change your address, you'll see what happens